src/components/PokemonDetails/PokemonDetails.js

```diff
@@ -24,16 +26,24 @@ class PokemonDetails extends Component {
     const { pokemonDetails, id } = this.state
     return(
       <>
+      <NavLink 
```

Comment:
can this just be a link since we arent giving it any styling?

Reply:
why are we not going to give any styling to the go back button?